wiki/Tournaments/otst!/2019/en.md

```diff
@@ -0,0 +1,222 @@
+   - 4 NoMod
+   - 2 Hidden
+   - 3 HardRock
+   - 2 DoubleTime
+   - 2 FreeMod
+   - 3 Tiebreakers
+3. Group stage will be round-robin, followed by a double elimination bracket.
+4. Maps will be no longer than 2 minutes drain time.
+5. The winner of `!roll` will ban and pick first.
+6. Teams get **one** ban in group stage, Round of 32 and Round of 16. The rest of final stage, teams will get **two** bans.
+7. Tiebreakers will be a BO3 instead of a single match.
+   - `!roll`, higher roll picks first during a tiebreaker.
+
+### Match Format
+
+1. In Group Stage, teams need to win 4 beatmaps to win a match. (Best-of-7)
+2. In the Round of 32 and the Round of 16, teams need to win 5 beatmaps to win a match. (Best-of-9)
+3. In Quarterfinals and the Semifinals, teams need to win 6 beatmaps to win a match. (Best-of-11)
+4. In the **Finals and Grand Finals**, teams need to win 7 beatmaps to win a match. (Best-of-13)
+   - There will be a bracket reset match if teams from losers bracket wins the first match.
+
+### Registration
+
+1. If a team member needs to be replaced after signing up, please contact a host **before** registration ends. No rosters can be modified after registration ends.
+2. Each member of participacing team must be in the Discord server to verify their participation. 
+3. If a player changes their name, they (or a team captain) needs to inform a tournament host or admin as soon as possible to avoid misunderstandings in a match.
+4. If a player exhibits suspicious behavior, including but not limited to suspected multi-accounting or cheating, their participation in the tournament might be prohibited, subject to tournament host discretion.
+5. Accounts that were recently unbanned or suspended within 12 months prior to February 9th will not be allowed to participate in otst!.
+
+### Scheduling
+
+*All times in the match schedules are in UTC.*
+
+1. Scheduling will be based on team timezones to avoid as many no-shows as possible.
+2. If your team wants to reschedule a match, please have yours and the opposing team agree on the new time and have either team captain submit proof of agreement in the Discord server's `#reschedule` text channel by **Thursday 23:59 UTC.**
+3. During the group stage, a 3v3 match could be arranged if some teammates cannot make it. The other team must be notified and agree to it too.
+   - **Remember**: 4v4s must be played in final stages.
+4. If you are certain that your team cannot make it to a match even after a reschedule, please tell a host and we will give you a loss for that match.
+   - Failure to do so will result in a penalty and teams may be exempt from future iterations or tournaments hosted by ![][flag_TH] [Seaweed](/users/5151214) or ![][flag_TH] [Trigonoculus](/users/7627013).
+
+### Match Proceedings
+
+- Referees have final decision in all matches so please respect their choices, but do not forget that they are human too and any disrespect **will result in penalties.**
```

Comment:
Any reason to have this as his own point?, probably better to remove the bullet and put this on italics